contentctl/output/templates/analyticstories_detections.j2

```diff
@@ -3,7 +3,7 @@
 
 {% for detection in objects %}
 {% if (detection.type == 'TTP' or detection.type == 'Anomaly' or detection.type == 'Hunting' or detection.type == 'Correlation') %}
-[savedsearch://{{APP_NAME}} - {{ detection.name }} - Rule]
+[savedsearch://{{ detection.get_conf_stanza_name(app) }}]
```

Comment:
Validate and get the full stanza name for the app, taking into account the APP it belongs to and ES-related length contstraints.